{"uuid": "7b2c2baa-1292-4727-81b8-020e73138c37", "children": ["9b156d96-c432-4954-a065-3a1edf14abb9", "4cfc73a7-5053-4cd4-86f6-7f908aceae0e", "34c4ae9d-8985-42d3-a3a3-b6f42fd6c6a0", "c187fb25-f58c-40b4-b1b0-1fd25da27de3", "8eb419bf-36e8-4251-a9b3-e9f550d2569b"], "befores": [{"name": "_Class__pytest_setup_class", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: Can not connect to the Service chromedriver\n\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/yangzhiqi/PycharmProjects/Selenium/venv/lib/python3.9/site-packages/_pytest/fixtures.py\", line 964, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/yangzhiqi/PycharmProjects/Selenium/venv/lib/python3.9/site-packages/_pytest/fixtures.py\", line 788, in call_fixture_func\n    res = next(it)\n  File \"/Users/yangzhiqi/PycharmProjects/Selenium/venv/lib/python3.9/site-packages/_pytest/python.py\", line 701, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"/Users/yangzhiqi/PycharmProjects/Selenium/venv/lib/python3.9/site-packages/_pytest/python.py\", line 632, in _call_with_optional_argument\n    func(arg)\n  File \"/Users/yangzhiqi/PycharmProjects/Selenium/TestCase/test_vue.py\", line 22, in setup_class\n    self.driver = webdriver.Chrome()\n  File \"/Users/yangzhiqi/PycharmProjects/Selenium/venv/lib/python3.9/site-packages/selenium/webdriver/chrome/webdriver.py\", line 70, in __init__\n    super(WebDriver, self).__init__(DesiredCapabilities.CHROME['browserName'], \"goog\",\n  File \"/Users/yangzhiqi/PycharmProjects/Selenium/venv/lib/python3.9/site-packages/selenium/webdriver/chromium/webdriver.py\", line 90, in __init__\n    self.service.start()\n  File \"/Users/yangzhiqi/PycharmProjects/Selenium/venv/lib/python3.9/site-packages/selenium/webdriver/common/service.py\", line 105, in start\n    raise WebDriverException(\"Can not connect to the Service %s\" % self.path)\n"}, "start": 1647331481856, "stop": 1647331512494}], "start": 1647331481856, "stop": 1647331512639}